backend/src/main/java/atlantbh/restaurants/models/Menu.java

```diff
@@ -0,0 +1,62 @@
+package atlantbh.restaurants.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+@Entity
+public class Menu extends BaseModel<Menu>{
```

Comment:
dont forget to format classess

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb